lib/mixlib/install/options.rb

```diff
@@ -167,7 +167,7 @@ def default_options
       end
 
       def validate_architecture
-        unless architecture.nil? || SUPPORTED_ARCHITECTURES.include?(architecture)
+        unless architecture.nil? || SUPPORTED_ARCHITECTURES.include?(Util.normalize_architecture(architecture))
```

Comment:
Is the input arch coming in from Ohai or from the detect_platform logic... or something else?